cylc/flow/commands.py

```diff
@@ -73,39 +79,43 @@
     CylcConfigError,
 )
 import cylc.flow.flags
+from cylc.flow.flow_mgr import get_flow_nums_set
 from cylc.flow.log_level import log_level_to_verbosity
 from cylc.flow.network.schema import WorkflowStopMode
 from cylc.flow.parsec.exceptions import ParsecError
 from cylc.flow.task_id import TaskID
-from cylc.flow.task_state import TASK_STATUSES_ACTIVE, TASK_STATUS_FAILED
-from cylc.flow.workflow_status import RunMode, StopMode
+from cylc.flow.task_state import (
+    TASK_STATUS_FAILED,
+    TASK_STATUSES_ACTIVE,
+)
+from cylc.flow.workflow_status import (
+    RunMode,
+    StopMode,
+)
 
-from metomi.isodatetime.parsers import TimePointParser
 
 if TYPE_CHECKING:
     from cylc.flow.scheduler import Scheduler
 
     # define a type for command implementations
-    Command = Callable[
-        ...,
-        AsyncGenerator,
-    ]
+    Command = Callable[..., AsyncGenerator]
+    _TCommand = TypeVar('_TCommand', bound=Command)
```

Comment:
What does `_TCommand` mean?

Reply:
The convention for generic types is to use a single letter like `T` or `S`, in this case I found out how to type this from some GitHub issue on python/typing or python/mypy. I seem to remember adding a link as a comment, but must have lost it